src/chrome/content/rules/Pando.xml

```diff
@@ -7,7 +7,7 @@
 		- pandoblog.com                (dm1.fastdomain.com)
 		- www.pandoblog.com            (dm1.fastdomain.com)
 -->
-<ruleset name="Pando (bad certs)" default_off="Wrong hostnames in certs.">
```

Comment:
Duplicate attribute. I'm going to skip commenting on any further duplicate attributes, since you can find them more efficiently by running ./test.sh.